infrastructure-playbooks/purge-docker-cluster.yml

```diff
@@ -437,6 +437,16 @@
     tags:
       remove_img
 
+  - name: find all osd_disk_prepare logs
+    find:
+      paths: "{{ ceph_osd_docker_run_script_path }}/ceph-osd-prepare-*.log"
+    register: osd_disk_prepare_logs
+
+  - name: ensure all osd_disk_prepare logs are removed
+    file:
+      path: "{{ item.path }}"
+      state: absent
+    with_items: "{{ osd_disk_prepare_logs.files }}"
```

Comment:
With file glob? So we can remove one task?

Reply:
jenkins test luminous-ansible2.4-purge_bluestore_osds_container